src/components/Categories.js

```diff
@@ -1,11 +1,9 @@
 import React from 'react';
 
-function Categories() {
-  return (
-    <div>
-      Under construction
-    </div>
-  );
-}
+const CategoryPage = () => (
+  <div>
+    <button type="button">CHECK STATUS</button>
+  </div>
```

Comment:
- [ ] This function should return a state stored message in your reducer `Under Construction`.